.github/workflows/build-deploy-dev.yml

```diff
@@ -5,42 +5,133 @@ on:
     types: [opened, reopened, synchronize, closed]
     branches:
       - main
+  workflow_dispatch:
+    inputs:
+      bumb:
+        description: "The type of semantic version increment to make. By default it is 'prerelease'."
+        required: false
+        default: "prerelease"
+        type: choice
+        options:
+          - major
+          - premajor
+          - minor
+          - preminor
+          - patch
+          - prepatch
+          - prerelease
+      preId:
+        description: "Identifier to be used to prefix version increments. By default it is 'prerelease' for pull request event and workflow dispatch will follow the last release tag prefix, if any."
+        required: false
+        type: choice
+        options:
+          -
+          - premajor
+          - preminor
+          - prepatch
+          - prerelease
```

Comment:
same goes here, if the default is prerelease, we can move it up.

Reply:
no, this case it depends on the trigger type